routing_model/index.html

```diff
@@ -832,14 +825,14 @@ <h2>Handling merges</h2>
   </div>
   <div class="mermaid">
       graph TB
-      ProcessMerge["ProcessMerge<br/>"]
+      ProcessMerge["ProcessMerge<br/>(shared state)"]
       EndRoutine["End of ProcessMerge<br/>(shared state)"]
       style ProcessMerge fill:#f9f,stroke:#333,stroke-width:4px
       style EndRoutine fill:#f9f,stroke:#333,stroke-width:4px
 
-      ProcessMerge -->  VoteOurMerge
-      VoteOurMerge["vote_for(<br/>ParsecOurMerge)"]
-      VoteOurMerge --> LoopStart
+      ProcessMerge -->  SendMergeRpc
+      SendMergeRpc["OUR_MERGE=true<br/>send_rpc(Merge)"]
```

Comment:
```
OUR_MERGE=true
```
I think this is unused now so could be removed
```
send_rpc(Merge)
```
Now it is done unconditionally, I think it would make sense to move it to `StartCheckAndProcessElderMergeChange`, that way it's the only place that needs to handle Merge RPCs

Reply:
I do not think so, this is an implementation detail of ProcessMerge, as this change shows, it did not need to happen there.

Reply:
Also, it is a different Merge RPC in the sense that we are sending that one, but receiving the other.
However the main point is that when we really process the merge RPC is when we check for consensus, so I'm not sure that it would achieve the goal of 'only place that needs to handle Merge RPCs'

Reply:
As discussed, we have a minor disagreement on this one, so we agreed to go with your suggestion.